ts/mcl.ts

```diff
@@ -176,7 +176,7 @@ export function parseG2(solG2: solG2): mclG2 {
 }
 
 function dump(sol: solG1 | solG2): string {
-    return `0x${sol.map(n => n.toString().slice(2)).join()}`;
+    return `0x${sol.map(n => n.toString().slice(2)).join("")}`;
```

Comment:
This is also my derp. `join` by default adds a comma separator which the previous implementation did not.